test/config/default/manager_image_patch.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - image: IMAGE_URL
```

Comment:
Add a comment telling user to replace this with their image

Reply:
Done.